cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -494,4 +494,13 @@
     /{,var/lib/snapd/}snap/core/*/usr/lib/snapd/snap-discard-ns rix,
     /var/lib/snapd/hostfs/{,var/lib/snapd/}snap/core/*/usr/lib/snapd/snap-discard-ns rix,
 
+    # As a special exception, allow HOME to be /var/lib/jenkins
+    /var/ r,
+    /var/lib/ r,
+    /var/lib/jenkins/ r,
+    /var/lib/jenkins/snap/{,*/,*/*/} rw,
```

Comment:
Why does snap-confine need write access to the snap subdirectory here? I looked at the other .is_optional directories and they don't need it.

Reply:
Ah, SNAP_USER_DATA. Can you add a comment for that?

Reply:
Yeah, this is so that snap-confine can create the user data directory.

Reply:
Done